### PR TITLE
Remove code in __init__ that's causing CI failures downstream

### PR DESCRIPTION
**Pull request by rosteen**
This code in __init__ is causing deprecation warnings (and thus CI failures) downstream when installing drizzlepac in CI. See for example https://github.com/spacetelescope/slitlessutils/actions/runs/9083277319/job/24961746510?pr=72. @pllim removed the same code from stsci.tools a while ago, and I'll be opening parallel PRs in a couple other packages.

### CODE REVIEW

**Comment by pllim** [comment]
Also see discussions in https://github.com/spacetelescope/stsci.image/pull/31

**Comment by rosteen** [comment]
The test failure looks unrelated.

**Comment by pllim** [comment]
numpy 1.20 is too old, and macos-latest is now ARM64, so I don't think the combo can find numpy wheel for that OS and tried to build numpy from source.

tl;dr -- yes, unrelated

**Review by zacharyburnett — APPROVED** [review]

**Comment by pllim** [comment]
I cannot merge this one, so please merge and then probably needs a release.

**Comment by pllim** [comment]
p.s. I just released we have a `stsci.image` and a `stsci.stimage`. 🤯
